config/cachedIconData.json

```diff
@@ -471,7 +471,7 @@
     },
     "2675:2064": {
       "id": "2675:2064",
-      "name": "ico-order",
+      "name": "ico-drag-indicator",
```

Comment:
This is OK. There was an error on the Figma file with two icons with `Order` as a name. The `cachedIconData` file was generated with two `order` icons so this PR is fixing that.